base/gocb_utils.go

```diff
@@ -134,14 +135,16 @@ func getRootCAs(caCertPath string) (*x509.CertPool, error) {
 
 		ok := rootCAs.AppendCertsFromPEM(caCert)
 		if !ok {
-			return nil, errors.New("Invalid CA cert")
+			return nil, errors.New("invalid CA cert")
 		}
 
 		return rootCAs, nil
 	}
 
-	// We're purposefully ignoring the error here Due to the fact that the main error case is that this call is not
-	// supported in Windows.
-	rootCAs, _ := x509.SystemCertPool()
+	rootCAs, err := x509.SystemCertPool()
+	if err != nil {
+		rootCAs = &x509.CertPool{}
```

Comment:
x509.CertPool contains a couple of maps that would panic if uninitialized when adding things to them.

https://play.golang.org/p/SXWLnrfnWKc

In this case that's probably not going to happen, but it seems safer to use `x509.NewCertPool()` anyway. We don't really know what gocb is doing under the hood with the CertPool we give them.